osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/BatchInserter.cs

```diff
@@ -164,7 +166,10 @@ private async Task run(HighScore[] scores)
                 }
                 catch (Exception e)
                 {
-                    throw new AggregateException($"Processing legacy score {highScore.score_id} failed.", e);
+                    if (throwOnFailure)
+                        throw new AggregateException($"Processing legacy score {highScore.score_id} failed.", e);
+
+                    Console.WriteLine($"Processing legacy score {highScore.score_id} failed.", e);
```

Comment:
```suggestion
                    Console.WriteLine($"Processing legacy score {highScore.score_id} failed: {e}");
```

pretty sure this isn't gonna do what you'd expect it to do otherwise

Reply:
Hmm, in my previous version i was throwing away the exception, which is actually nicer in logging.

So... it does what I want, just in a weird way 😅

Reply:
alright well then 
```suggestion
                    Console.WriteLine($"Processing legacy score {highScore.score_id} failed.");
```
i guess